mlflow/models/__init__.py

```diff
@@ -42,9 +48,15 @@ def add_flavor(self, name, **params):
         self.flavors[name] = params
         return self
 
+    def to_dict(self):
+        return self.__dict__
```

Comment:
Was looking at the [Python docs](https://docs.python.org/3/library/stdtypes.html#object.__dict__) for this, will/does this also include attributes of a `Model` instance that are functions etc?

Reply:
Good question, it does not include methods, but it does include whatever you assign to the objects even if it was a function. Any weird objects such as lambdas may fail to serialize to json, however, we use the same mechanism to export Model to yaml format, so such Model would not be loggable anyways.